media-sound/teamspeak-client-bin/teamspeak-client-bin-3.0.16.ebuild

```diff
@@ -20,6 +20,7 @@ IUSE="alsa pulseaudio"
 REQUIRED_USE="|| ( alsa pulseaudio )"
 
 RDEPEND="
+	dev-libs/quazip[qt5]
```

Comment:
I was already thinking about adding a system-quazip USE flag in case somebody needs quazip[qt4] installed